src/FileServiceManager/FileService.Tests/FileSystemJsonConfiguratorTest.cs

```diff
@@ -0,0 +1,95 @@
+using FileService.DAL;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using System.Threading.Tasks;
+using System.IO;
+using Microsoft.Extensions.Options;
+using FileService.DAL.Storages.Options;
+using FileService.Models;
+using System;
+using System.Linq;
+
+
+namespace FileService.Tests
+{
+    [TestClass]
+    public class FileSystemJsonConfiguratorTest
+    {   
+        static IFileSystemConfigurator sut;
+        static IOptions<StorageOption> option;
+        static int newConfigurationCompanyID = 1001;
+        static Guid newConfigurationGuid = Guid.NewGuid();
+
+        static int companyIdToDeleteConfigFrom = 2001;
+        static Guid guidToDeleteConfigFrom = Guid.Parse("cb766b03-9553-4caa-a588-9493870a2bad");
+
+        [ClassInitializeAttribute]
+        public static async Task Setup(TestContext ctx)
+        {
+            option = Options.Create(new StorageOption {Path = Path.Combine(Directory.GetCurrentDirectory(), "json1").Replace("bin\\Debug\\netcoreapp5.0\\", "")});
+            sut = new FileSystemJsonConfigurator(option);
+
+            await sut.AddConfigurationAsync(companyIdToDeleteConfigFrom, new FileMetadata() {Id = guidToDeleteConfigFrom});
+        }
+
+        [TestMethod] 
+        public async Task Test1_AddConfugration_Should_Pass_If_File_Is_Valid() 
```

Comment:
what this test is validating ?

Reply:
checks the flow for the case, when adding file bundled to the company for the first time.

Reply:
I don't see here any validation, shouldn't we assert in something here ?

Reply:
My apologies, missed that part